packages/openassistants/openassistants/eval/checks/function_selection.py

```diff
@@ -0,0 +1,22 @@
+from openassistants.eval.v2 import (
```

Comment:
I'm having difficulty identifying where `v2` comes from here, I don't see a `v2` in `packages/openassistants/openassistants/eval/`

When I run from poetry shell:

```
from openassistants.eval.v2 import BaseResponseChecker
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'openassistants.eval.v2'
```

This looks like a bug.

Reply:
same as below